spikeextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -416,7 +420,9 @@ def add_electrical_series(recording, nwbfile, metadata):
         Auxiliary static method for nwbextractor.
         Adds traces from recording object as ElectricalSeries to nwbfile object.
         """
-        # ElectricalSeries
+        if 'Ecephys' not in metadata:
```

Comment:
I think the structure of this metadata dict is going to change

Reply:
This particular change is mostly an error-check if the user ever calls NwbRecordingExtractors.add_electrical_series directly in isolation. If called from the NwbRecordingExtractor.write_recording, the base keys are already created by the surrounding add functions.

Do you mean the passed metadata dictionary is soon to require revamping to adhere to the json schema from the new converter format? If so, then yeah we'll have to go through some of these helper functions to adjust the defaults.

Reply:
> Do you mean the passed metadata dictionary is soon to require revamping to adhere to the json schema from the new converter format?
Yes, that's what I mean

Reply:
So should we modify this or can we merge as it is? We can always modify it later :)

Reply:
yeah lets proceed